biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -845,6 +882,12 @@ subroutine fire_litter_fluxes(currentSite, cp_target, new_patch_target, patch_si
     real(r8) :: dead_tree_density    ! no trees killed by fire per m2
     reaL(r8) :: burned_litter        ! amount of each litter pool burned by fire.  kgC/m2/day
     real(r8) :: burned_leaves        ! amount of tissue consumed by fire for grass. KgC/individual/day
```

Comment:
if this "burned_leaves" is specific to grasses and differs from "leaf_burn_frac" we should update the names of both to be more specific